.gitignore

```diff
@@ -80,4 +80,4 @@ packages/*/debug/
 /packages/*/.rpt2_cache
 
 ## for john to poke at
-test.html
```

Comment:
we shouldnt stop ignoring this file.

Reply:
Yes, I have been testing with that. I remember to add it back before merging

Reply:
for sure.

the point of the file is to have something to use for local testing that doesn't ever get committed.